cluster-autoscaler.spec

```diff
@@ -85,11 +85,10 @@ have a place to run and there are no unneeded nodes.
 
 %prep
 %if 0%{do_prep}
-%setup -n kubernetes-autoscaler-cluster-autoscaler-%{version}
-mkdir `pwd`/../autoscaler
-mv * `pwd`/../autoscaler
-mkdir -p `pwd`/go/src/k8s.io
-mv `pwd`/../autoscaler `pwd`/go/src/k8s.io
+%setup -n kubernetes-autoscaler-cluster-autoscaler-%{commit}
```

Comment:
above is shortcommit and here it is commit?

Reply:
When you download `https://github.com/openshift/kubernetes-autoscaler/archive/e9bb914c988215b7e834dd3bb09a768bc18682ba/kubernetes-autoscaler-e9bb914.tar.gz`, the directory extracted from the tarball is `kubernetes-autoscaler-cluster-autoscaler-e9bb914c988215b7e834dd3bb09a768bc18682ba`.